CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,87 @@
+- The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+- Trolling, intentionally spreading misinformation, insulting/derogatory
+  comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+- Abuse of the reporting process to intentionally harass or exclude others
+- Advocating for, or encouraging, any of the above behavior
+- Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, or to ban temporarily or permanently any
+contributor for other behaviors that they deem inappropriate, threatening,
+offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting us anonymously through
+[the discord](https://discord.gg/trashdev). All complaints will be
+reviewed and investigated and will result in a response that is deemed necessary
+and appropriate to the circumstances. The project team is obligated to maintain
+confidentiality with regard to the reporter of an incident.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+If you are unsure whether an incident is a violation, or whether the space where
+the incident took place is covered by our Code of Conduct, **we encourage you to
+still report it**. We would prefer to have a few extra reports where we decide
+to take no action, than to leave an incident go unnoticed and unresolved that
+may result in an individual or group to feel like they can no longer participate
+in the community. Reports deemed as not a violation will also allow us to
+improve our Code of Conduct and processes surrounding it. If you witness a
+dangerous situation or someone in distress, we encourage you to report even if
+you are only an observer.
```

Comment:
suggestion: could also add the following.
- "we also encourage you to report it to the appropriate authorities if you think the situation can have severe consequences" 
- end with something like "do whatever you think is best for the situation". 

imo we shouldn't "only" tell them to report it to us. they should do the best for the situation.

Reply:
I agree whole heartedly.

But at the same edge I worry about going into that fine-grain of detail? Like if someone in the github issues is saying they're going to hunt trash down and kill him, I'd hope whoever saw that wouldn't need to have read "contact the proper authorities" in our code of conduct do do the right thing here?

Reply:
>at the same edge I worry about going into that fine-grain of detail?

Yep! I worried about this too. It's common sense to act according to the best of the situation. I'll mark this as resolved @jhechtf. This could improve over time.